src/main/kotlin/no/nav/tiltakspenger/saksbehandling/meldekort/domene/MeldekortBehandletAutomatisk.kt

```diff
@@ -115,5 +119,10 @@ fun Sak.opprettAutomatiskMeldekortBehandling(
         beregning = MeldekortBeregning(beregninger),
         type = MeldekortBehandlingType.FØRSTE_BEHANDLING,
         status = MeldekortBehandlingStatus.AUTOMATISK_BEHANDLET,
-    ).right()
+        simulering = null,
+    )
+    return simuler(meldekortBehandletAutomatisk).mapLeft {
```

Comment:
Hvorfor ønsker vi simulering for det som behandles automatisk? Er ikke simuleringen for at saksbehandler skal oppdage mulige problemer?

Reply:
Fikk svar på slack :)